packages/forms-web-app/src/controllers/appellant-submission/check-answers.js

```diff
@@ -1,8 +1,12 @@
 const { getDepartmentFromId } = require('../../services/department.service');
 const { VIEW } = require('../../lib/views');
+const logger = require('../../lib/logger');
 
 exports.getCheckAnswers = async (req, res) => {
   const { appeal } = req.session;
+
+  logger.info(appeal);
+
```

Comment:
couldnt work out how to look in my local db so ^ to get an id out.. leaving it in case other people are in a similar position while looking at this, obviously i'm not proposing it goes into master..